docs/specs/git.yml

```diff
@@ -408,55 +408,20 @@ outputs:
       "gitcommit": "https://github.com/testowner/testrepo/blob/e662dbc7c5e582a14b863d6a6a82e495d6272f91/docs/index.md",
     }
 ---
-# Show git contribution info for vsts repo
+# Show (original) content git (branch/commit) url for vsts
 repos:
-  https://docascode.visualstudio.com/contribution/_git/a:
-    - author: Charlie
-      email: charlie@contoso.com
-      files:
-        docs/index.md: |
-          ---
-          author: charlie
-          ---
-          Osmond creates this. Bob updates this. Charlie becomes author.
-    - author: Bob 
-      email: bob@contoso.com
-      files:
-        docs/index.md: |
-          Osmond creates this. Bob updates this.
```

Comment:
>"gitcommit":"https://docascode.visualstudio.com/git-url/_git/a/?path=docs/index.md&branch=GB73d1bc95f9df03e553c39cf8c685df09af8d5297 [](start = 6, length = 132)

[https://ceapex.visualstudio.com/Engineering/_git/Docs.DocFX.Impact?path=%2Fpackage.json&version=GBdfc139139a2a9ea996ff9b25a49ca7e812769a0a](https://ceapex.visualstudio.com/Engineering/_git/Docs.DocFX.Impact?path=%2Fpackage.json&version=GBdfc139139a2a9ea996ff9b25a49ca7e812769a0a)

Tried the above link, it didn't work. And there's an extra trailing '\' after repo name(a) ?

Reply:
It should be version=G~~B~~**C**dfc139139a2a9ea996ff9b25a49ca7e812769a0a`

Reply:
already changed that.